packages/core/src/components/PageTabs/PageTabs.tsx

```diff
@@ -24,12 +25,23 @@ export const PageTabs: FC<TabsProps> = (props) => {
   const firstMatch = useMemo(() => findMatchPath(location.pathname, props.tabs), []);
   const [activeTab, setActiveTab] = useState(firstMatch);
   const items = useMemo(() => props.tabs.map(({ Title }) => Title), [props.tabs]);
+
```

Comment:
change this line to return items with {Component, disabled: boolean}

```
const items = useMemo(() => props.tabs.map(({ Title, disabled }) => ({Component, disabled}), [props.tabs]);
```

and remove the disabledTabs props. then change in the render function;

```
      {items.map(( { Component, disabled } , idx) => (
          <Tab
            className={classNames(`${clsPrefix}-item`, {
              [`${clsPrefix}-item-active`]: activeTab === idx,
            })}
            idx={idx}
            onClick={onTabChange}
            disabled={disabled}
            key={idx}
          >
              <Component />
          </Tab>
        ))}
```